app/views/service/_svcs_show.html.haml

```diff
@@ -28,7 +28,7 @@
             .form-horizontal.static
               .form-group
                 .col-md-9
-                  - htm = @job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'")
+                  - htm = @job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'").gsub(/{{/, '\{\{').gsub(/}}/, '\}\}')
```

Comment:
Each `gsub` call is going to return a copy of the original string, since these can be large output it would make more sense to do separate `gsub!` calls so the substitutions happens without duplicating the string.

There might also be a way to do this in one `gsub!` call.  For example:
`'hello'.gsub(/[eo]/, 'e' => 3, 'o' => '*')    #=> "h3ll*"` (See https://ruby-doc.org/core-2.1.4/String.html)